js/common/bootstrap/errors.js

```diff
@@ -366,43 +367,5 @@
 
   // For compatibility with <= 3.3
   internal.errors.ERROR_ARANGO_COLLECTION_NOT_FOUND = internal.errors.ERROR_ARANGO_DATA_SOURCE_NOT_FOUND;
-
-  //arg1 can be a code or error object
-  internal.throwArangoError = function (arg1, message, httpCode) {
```

Comment:
I'm not sure where this came from. I rebuilt with maintainer mode and this code vanished.